src/services/shortLink/test/index.test.ts

```diff
@@ -0,0 +1,552 @@
+      });
+
+      it('Success when author', async () => {
+        const { item } = await mockPostData({ author: actor });
+        const payload = mockShortLinkPayload({ item_id: item.id });
+        const post = await injectPost(app, payload);
+        expect(post.statusCode).toEqual(StatusCodes.OK);
+        expect(post.json().alias).toEqual(MOCK_ALIAS);
+
+        const response = await injectDelete(app, MOCK_ALIAS);
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json().alias).toEqual(MOCK_ALIAS);
+      });
+    });
+  });
+
+  describe('GET /short-links/items/:itemId', () => {
+    const numberOfShortLinks = 5;
+
+    beforeEach(async () => {
+      ({ app, actor } = await build());
+      bob = await saveMember(BOB);
+      cedric = await saveMember(CEDRIC);
+      anna = await saveMember(ANNA);
+
+      ({ item } = await mockPostData({
+        author: actor,
+        member: bob,
+        permission: PermissionLevel.Admin,
+      }));
+
+      await saveMembership({ item, member: anna, permission: PermissionLevel.Write });
+
+      for (let i = 0; i < numberOfShortLinks; i++) {
+        const response = await injectPost(
+          app,
+          mockShortLinkPayload({ item_id: item.id, alias: MOCK_ALIAS + i }),
+        );
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json().alias).toEqual(MOCK_ALIAS + i);
+      }
+    });
+
+    it('Not found if get short links with invalid item id', async () => {
+      const response = await injectGetAll(app, MOCK_ITEM_ID);
+      expect(response.statusCode).toEqual(StatusCodes.NOT_FOUND);
+    });
+
+    describe('Unauthorized and forbidden', () => {
+      const attemptFailure = async (statusCode: StatusCodes) => {
+        const response = await injectGetAll(app, item.id);
+        expect(response.statusCode).toEqual(statusCode);
+      };
+
+      it('Unauthorized when not connected', async () => {
+        logOut(app);
+        await attemptFailure(StatusCodes.UNAUTHORIZED);
+      });
+
+      it('Forbidden if unauthorized', async () => {
+        ({ app } = await build({ member: cedric }));
+        await attemptFailure(StatusCodes.FORBIDDEN);
+      });
+
+      it('Forbidden if not admin', async () => {
+        ({ app } = await build({ member: anna }));
+        await attemptFailure(StatusCodes.FORBIDDEN);
+      });
+    });
+
+    describe('Success', () => {
+      it('Success if item exist', async () => {
+        const { item } = await mockPostData({ author: actor });
+        const response = await injectGetAll(app, item.id);
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json()).toEqual([]);
+      });
+
+      const attemptSuccess = async () => {
+        const response = await injectGetAll(app, item.id);
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json()).toHaveLength(numberOfShortLinks);
+      };
```

Comment:
Don't add a function in the middle of the describe scope. I think it's okay to c/p this code twice.